app/src/fdroid/java/com/duckduckgo/app/di/StoreReferralModule.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2019 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.di
+
+import android.content.Context
+import com.duckduckgo.app.referral.*
+import com.duckduckgo.app.statistics.AtbInitializerListener
+import com.duckduckgo.referral.FDroidReferrerStateListener
+import dagger.Module
+import dagger.Provides
+import dagger.multibindings.IntoSet
+import javax.inject.Singleton
+
+@Module
+class StoreReferralModule {
```

Comment:
we have now two `StoreRefferralModule`, only one will be used based on the variant being built